content/docs/components/serving/istio.md

```diff
@@ -11,21 +11,26 @@ quota, rollout and A/B testing.
 We assume Kubeflow is already deployed in the `kubeflow` namespace.
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/kubeflow/kubeflow/master/dependencies/istio/install/crds.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubeflow/kubeflow/master/dependencies/istio/install/istio-noauth.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubeflow/kubeflow/master/dependencies/istio/kf-istio-resources.yaml
+$ curl -L https://git.io/getLatestIstio | ISTIO_VERSION=1.3.3 sh -
+$ export PATH=$HOME/istio-1.3.3/bin:$PATH
+$ for i in $HOME/install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
+```
+Open `$HOME/install/kubernetes/istio-demo.yaml` and change *istio-ingressgateway* service from *LoadBalancer* to *NodePort*. Then run:
```

Comment:
Why need this?

Reply:
This is needed because we don't need the istio-ingress gateway, to expose services on publicly accessible IP addresses by default because its unsafe as mentioned by @jlewi in [#664](https://github.com/kubeflow/examples/pull/664). Plus in the default page it has been written that there has been some modification done in ingress gateway to change it from LoadBalancer to Nodeport as mentioned [here](https://www.kubeflow.org/docs/components/serving/istio/)